pyk/src/pyk/cli/args.py

```diff
@@ -206,6 +206,7 @@ class KompileOptions(Options):
     bison_lists: bool
     no_exc_wrap: bool
     outer_parsed_json: bool
+    ignore_warnings: list[str] | None
```

Comment:
I'm not a 100% sure in this, but consider

```suggestion
    ignore_warnings: list[str]
```

This makes it consitent with `ccopts`.

@nwatson22, is this the right convention to follow?

Reply:
Yes, I'd go without the `None` type option unless there is a situation where it could actually be set to `None`, which there shouldn't be as far as I know since you have `[]` as the default.